_rules/auto-update-text-efbfc7.md

```diff
@@ -0,0 +1,336 @@
+---
+id: efbfc7
+name: Auto-updating text content can be paused or stopped
+rule_type: atomic
+
+description: |
+  This rule checks that there are mechanisms to pause or stop the auto-updating of text content
+
+accessibility_requirements: # Remove whatever is not applicable
+  wcag20:2.2.2: # Pause, Stop, Hide (A)
+    forConformance: true
+    failed: not satisfied
+    passed: further testing needed
+    inapplicable: further testing needed
+
+input_aspects:
+  - DOM Tree
+
+authors:
+  - Carlos Duarte
+---
+
+## Applicability
+
+The rule applies to any [text node][] that is [visible][] and a [child][] in the [flat tree][] of an HTML element, whose content changes through an action that was not initiated by the user, anytime after the [readiness][document readiness] of the document the [text node][] belongs to is equal to "complete".
```

Comment:
what about nodes that are not part of the accessible tree?

Reply:
I might be misinterpreting the SC, but my understanding is that it applies to visible content, whether or not it is part of the accessibility tree. The understanding document states that:
- "Moving, blinking and scrolling" refers to content in which the visible content conveys a sense of motion
- The requirements for moving, blinking and scrolling content and for auto-updating content are the same (except for the possibility of allowing users to control the frequency of updates and the 5 second rule)
So my understanding is that the content must be visible, independent of being in the accessibility tree. 
What is everyone's opinion on this?